nilearn/plotting/tests/test_displays.py

```diff
@@ -137,6 +137,12 @@ def test_annotations():
                          scale_size=2.5,
                          scale_units='cm',
                          scale_loc=3)
+
+
+def test_position_annotation_with_decimals():
+    # Test of decimals position annotation with precision of 2
+    orthoslicer = OrthoSlicer(cut_coords=(0, 0, 0))
+    orthoslicer.annotate(positions=True, decimals=2)
```

Comment:
this test doesn't seem to check the number of decimals is actually respected?

Reply:
Will make the new test to check number of decimals.